actions/global_actions.jsx

```diff
@@ -47,7 +47,7 @@ export function emitChannelClickEvent(channel) {
     async function userVisitedFakeChannel(chan, success, fail) {
         const state = getState();
         const currentUserId = getCurrentUserId(state);
-        const otherUserId = Utils.getUserIdFromChannelName(chan);
+        const otherUserId = Utils.getUserIdFromChannelName(chan, currentUserId);
```

Comment:
This looks like a rebase error, because probably I pass the userId's so I can avoid the calling of ```getState``` in ```export function getUserIdFromChannelId(channelId, currentUserId = getCurrentUserId(store.getState())) {
``` in utils.jsx